package.json

```diff
@@ -57,7 +57,7 @@
   "dependencies": {
     "@atlaskit/icon": "21.1.0",
     "@atlaskit/media-core": "32.2.0",
-    "@material-ui/core": "4.12.3",
+    "@material-ui/core": "4.11.4",
```

Comment:
why this downgrade?

Reply:
it is required with the version of cozy-ui we use